jena-fuseki2/jena-fuseki-ui/src/views/dataset/Edit.vue

```diff
@@ -24,16 +24,20 @@
           <b-card-header header-tag="nav">
             <Menu :dataset-name="datasetName" />
           </b-card-header>
-          <b-card-body>
+          <b-card-body v-if="!this.services['gsp-rw'] || this.services['gsp-rw'].length === 0">
+            <b-alert show variant="warning">No service for Graph Store Protocol configured</b-alert>
```

Comment:
Uses may not associate GSP with thier task of adding data. Maybe working like:

```suggestion
            <b-alert show variant="warning">No service for adding data available. The Graph Store Protocol service should be configured to allow adding data.</b-alert>
```

Reply:
Will copy the text in Upload.vue too. Thanks!